### DISCUSSION TITLE
How does the IGNORE_USERS parameter work?

### DISCUSSION THREAD

**Discussion by dariusz-rusin** [github-community-projects/issue-metrics#363] [General]
Is it intentional that the PR list still shows PRs from ignored users? Does this mean that when calculating metrics, these pointed out users are actually ignored despite the fact that their PRs are reported in the list?

Workflow:
![Screenshot 2024-09-11 at 15 07 43](https://github.com/user-attachments/assets/43a4ba8e-e4bd-433e-a654-98a3e1379410)

Result:
![Screenshot 2024-09-11 at 15 07 06](https://github.com/user-attachments/assets/b97293fe-1b79-48dc-bf09-74793ef04c9d)

**Comment by zkoppert** [comment]
Refreshing my memory on this by looking at the code. The design of `ignore_users` removes it from the calculation of metrics but does not remove it from the list that prints. Even though it changes existing functionality we should probably open a pull request to pass the ignore users list to the markdown writing function so it can filter out those users. 

Does that make sense? Overall, its expected that you see them in the list but they were not included in the calculation of the overall metrics. By overall metrics, I mean the ones that look like this:

| Metric | Average | Median | 90th percentile |
| --- | --- | --- | ---: |
| Time to first response | 7:30:09 | 0:46:19 | 20:11:41 |
| Time to close | 20:15:34 | 7:01:18 | 2 days, 22:10:29 |

| Metric | Count |
| --- | ---: |
| Number of items that remain open | 1 |
| Number of items closed | 90 |
| Number of most active mentors | 0 |
| Total number of items created | 91 |

**Reply by zkoppert** [reply]
I filed a bug report for this: https://github.com/github/issue-metrics/issues/365

**Reply by dariusz-rusin** [reply]
Thanks for the information. This is exactly what I suspected. In my opinion, the result will be clearer if there is a list of those PRs that are taken for calculation.

**Reply by zkoppert** [reply]
Its fixed! https://github.com/github/issue-metrics/releases/tag/v3.12.0
